### ISSUE TITLE
github login page and reset password, unable to see text due to same dark color

### ISSUE THREAD

**Issue by Sagar19RaoRane** [darkreader/darkreader#5534]
# Bug Report

## Describe the bug

 github login page and reset password, unable to see text due to same dark color
## System Information

- OS: 
- Browser: 
- Dark Reader Version:

**Comment by Gusted** [comment]
Hi @Sagar19RaoRane!

Do you have any other extensions? I couldn't reproduce the bug.

**Comment by Sagar19RaoRane** [comment]
No, I don't have any other extension installed. i was using this github account for long time. suddenly though to use it. hence i came across this bug while i was resetting the password, the same happened on login page

**Comment by Gusted** [comment]
Hmm I don't see a clear reason how it can happen is it 100% reproduce-able?

**Comment by do-wie-ching** [comment]
# I couldn't reproduce the bug on Chrome.
![GitHub Logo](https://github.com/do-wie-ching/screenshots/blob/main/Screenshot%202021-09-23%20002756.png?raw=true)
* OS: windows
* Browser: Chrome 94
* Dark Reader Version: 4.9.36
